config-xml/addon/src/main/java/org/jboss/windup/config/parser/XMLRuleProviderLoader.java

```diff
@@ -201,6 +204,6 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
     }
 
     private boolean pathMatchesNamePattern(Path file) {
-        return file.getFileName().toString().toLowerCase().endsWith("." + XML_RULES_EXTENSION);
+        return file.getFileName().toString().toLowerCase().endsWith("." + XML_RULES_WINDUP_EXTENSION) || file.getFileName().toString().toLowerCase().endsWith("." + XML_RULES_RHAMT_EXTENSION);
```

Comment:
This is not readable much, can you format according to our format style? We have 150 columns on line at max.